core/src/game/game_feature/game_feature.service.ts

```diff
@@ -19,6 +19,9 @@ export class GameFeatureService {
     ) {}
 
     async featureIsEnabled(code: FeatureCode, gameId: number, organizationId: number): Promise<boolean> {
+        console.log({
```

Comment:
Need this?

Reply:
https://github.com/SprocketBot/sprocket/pull/225/commits/d1c94d09428cd8b2443b3a6e9adbc375d7ab0b9a